files/scripts/update-cluster.sh

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env bash
+set -eu
+
+function patchCluster {
+  local configLocation=$1
+  local authToken=$2
+  echo "Patching cluster"
+  waitService "http://graphdb-cluster-proxy:7200/proxy/ready" "$authToken"
+  curl -o patchResponse.json -isSL -m 15 -X PATCH  --header "Authorization: Basic ${authToken}" --header 'Content-Type: application/json' --header 'Accept: application/json' -d @"$configLocation" 'http://graphdb-cluster-proxy:7200/rest/cluster/config'
+    if grep -q 'HTTP/1.1 200' "patchResponse.json"; then
+      echo "Patch successful"
+    elif grep -q 'Cluster does not exist.\|HTTP/1.1 412' "patchResponse.json" ; then
+      echo "Cluster does not exist"
+    else
+      echo "Cluster patch failed, received response:"
+      cat patchResponse.json
+      echo
+      exit 1
+    fi
+}
+
+function removeNodes {
+  local expectedNodes=$1
+  local authToken=$2
+  local namespace=$3
+  local currentNodes=$(getNodeCountInCurrentCluster "$authToken")
+  local nodes=""
+# if there is no cluster or current nodes are less or equal to expected so no need to remove more, exit
+  if [ "$currentNodes" -lt 2 ] || [ "$currentNodes" -le "$expectedNodes" ]; then
+    echo "No scaling required"
```

Comment:
write more detailed messages, it's good to know that the script went into a cluster downscaling, how many nodes are currently available and how many are wished. Also if there's a cluster or not, if a cluster is deleted (in the case below), etc. We'll have to debug what happened in case of a bug based on logs, so it's better to make them more detailed

Reply:
Added massages for current and wished nodes, if it's scaling up, down, deleting a cluster or not interested in scaling.